tests/functional/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -225,3 +233,126 @@ def test_non_ocs_taint_and_tolerations(self):
                 resource_count=count * replica_count,
             ), "New OSDs failed to reach running state"
             check_ceph_health_after_add_capacity(ceph_rebalance_timeout=2500)
+
+    @skipif_ocs_version("<=4.15")
+    def test_new_custom_taint_and_tolerations(self, nodes):
```

Comment:
I was reviewing this PR https://github.com/red-hat-storage/ocs-ci/pull/9808/files, Please correct if i misunderstood anything here, There is a similar testcase and its being skipped from 4.16 https://github.com/red-hat-storage/ocs-ci/blob/master/tests/functional/z_cluster/nodes/test_non_ocs_taint_and_toleration.py, I understand that the way of adding toleration is changed now, But steps and validations are similar. Can we convert this into a single test, test >=4.16 test_old <4.16